exports.js

```diff
@@ -14,6 +14,8 @@ module.exports = {
         'cloudfrontLoggingEnabled'      : require(__dirname + '/plugins/aws/cloudfront/cloudfrontLoggingEnabled.js'),
         'cloudfrontWafEnabled'          : require(__dirname + '/plugins/aws/cloudfront/cloudfrontWafEnabled.js'),
 
+        'plaintextParameters'           :  require(__dirname + '/plugins/aws/cloudformation/plaintextParameters.js'),
```

Comment:
the file is named "plainTextParameters" notice the uppercase T. This is also why the test is failing